lib/mozart_fetcher/fake_origin.ex

```diff
@@ -0,0 +1,18 @@
+defmodule MozartFetcher.FakeOrigin do
+  use Plug.Router
+
+  plug Plug.Parsers, parsers: [:json],
+                     pass:  ["text/*"],
+                     json_decoder: Poison
+
+  plug :match
+  plug :dispatch
+
+  get "/success" do
+    send_resp(conn, 200, ~s({"head":[],"bodyInline":"<DIV id=\\"site-container\\">","bodyLast":[]}) )
+  end
+
+  get "/timeout" do
+    :timer.sleep(3100)
```

Comment:
we could have a very short timeout for the test env right?

Reply:
I think that might be the way to go yeah. I'll see how easy this is to implement. I hope just a small thing in application.ex